convene-web/db/schema.rb

```diff
@@ -44,6 +44,18 @@
     t.index ["room_id"], name: "index_furniture_placements_on_room_id"
   end
 
+  create_table "identities", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name"
+    t.string "pronoun"
+    t.uuid "person_id"
+    t.uuid "space_id"
```

Comment:
Maybe instead of `space_id` on `Identity`, we have a `profile_id` on `SpaceMembership` so that an Identity can be used in multiple Spaces?

Reply:
Good, let me look into it and think it thru. 👍

Reply:
Something to think about, move avatar_url to Identity or have avatar_url on Person as a fallback?